charts/bindplane/templates/bindplane-jobs.yaml

```diff
@@ -0,0 +1,447 @@
+              value: sendgrid
+            - name: BINDPLANE_SEND_GRID_API_TOKEN
+              value: {{ .Values.email.sendgrid.token }}
+            {{- end }}
+            - name: BINDPLANE_PORT
+              value: "3001"
+            - name: BINDPLANE_PROMETHEUS_ENABLE
+              value: "true"
+            - name: BINDPLANE_PROMETHEUS_ENABLE_REMOTE
+              value: "true"
+            - name: BINDPLANE_PROMETHEUS_HOST
+              {{- if .Values.prometheus.remote }}
+              value: {{ .Values.prometheus.host }}
+              {{- else }}
+              value: {{ include "bindplane.fullname" . }}-prometheus
+              {{- end }}
+            - name: BINDPLANE_PROMETHEUS_PORT
+              value: "{{ .Values.prometheus.port }}"
+            {{- if .Values.prometheus.queryPathPrefix }}
+            - name: BINDPLANE_PROMETHEUS_QUERY_PATH_PREFIX
+              value: {{ .Values.prometheus.queryPathPrefix }}
+            {{- end }}
+            {{- if and (.Values.prometheus.remoteWrite.host) (.Values.prometheus.remoteWrite.port) }}
+            - name: BINDPLANE_PROMETHEUS_REMOTE_WRITE_HOST
+              value: {{ .Values.prometheus.remoteWrite.host }}
+            - name: BINDPLANE_PROMETHEUS_REMOTE_WRITE_PORT
+              value: "{{ .Values.prometheus.remoteWrite.port }}"
+            {{- end }}
+            - name: BINDPLANE_PROMETHEUS_REMOTE_WRITE_ENDPOINT
+              value: {{ .Values.prometheus.remoteWrite.path }}
+            - name: BINDPLANE_PROMETHEUS_AUTH_TYPE
+              value: {{ .Values.prometheus.auth.type }}
+            {{- if eq .Values.prometheus.auth.type "basic" }}
+            - name: BINDPLANE_PROMETHEUS_AUTH_USERNAME
+              value: {{ .Values.prometheus.auth.username }}
+            - name: BINDPLANE_PROMETHEUS_AUTH_PASSWORD
+              value: {{ .Values.prometheus.auth.password }}
+            {{- end }}
+            {{- if .Values.prometheus.tls.enable }}
+            - name: BINDPLANE_PROMETHEUS_ENABLE_TLS
+              value: "true"
+            - name: BINDPLANE_PROMETHEUS_TLS_SKIP_VERIFY
+              value: "{{ .Values.prometheus.tls.insecure }}"
+            {{- if .Values.prometheus.tls.secret.caSubPath }}
+            - name: BINDPLANE_PROMETHEUS_TLS_CA
+              value: /prometheus-ca.crt
+            {{- end }}
+            {{- if .Values.prometheus.tls.secret.crtSubPath }}
+            - name: BINDPLANE_PROMETHEUS_TLS_CERT
+              value: /prometheus-client.crt
+            {{- end }}
+            {{- if .Values.prometheus.tls.secret.keySubPath }}
+            - name: BINDPLANE_PROMETHEUS_TLS_KEY
+              value: /prometheus-client.key
+            {{- end }}
+            {{- end }}
+            {{- if len .Values.extraEnv }}
+            {{- toYaml .Values.extraEnv | nindent 12 }}
+            {{- end }}
+          {{- with .Values.jobs.resources }}
+          resources:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
```

Comment:
Jobs specific resource configuration. In a highly scaled environment, it is possible the main pods will have low cpu and memory values, so we want the jobs pod to have its own configuration.